src/App.vue

```diff
@@ -26,17 +26,19 @@
   </div>
 </template>
 
-<script>
+<script lang="ts">
 export default {
   name: 'TIDO',
 };
 </script>
 
-<script setup>
+<script setup lang="ts">
 import {
   computed, inject, onMounted, ref,
 } from 'vue';
 import { useStore } from 'vuex';
+import { useConfigStore } from './stores/config';
```

Comment:
The path should use `@` instead of relative paths

Reply:
yes, changed the paths